plugins/status/configs/.rr-status-init.yaml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 
 rpc:
-  listen: tcp://127.0.0.1:6005
+  listen: tcp://127.0.0.1:6007
```

Comment:
I will make the port common to the three configs. It used to be the same for two configs.

Reply:
All tests should have (ideally) their own ports for the RPC and other transports, due to the actions slow ports deallocation between the tests and `port already in use` errors.

Reply:
Fixed